crane-cloud/backend
# Ft app metrics

# Description

This PR advances the return perimeters on the apps endpoint for admin and it enables saving app statuses  

## Type of change



## Trello Ticket ID
https://trello.com/c/Js6ypYMs

## How Can This Be Tested?

Run this branch and tests the admin app filters, and also try to fetch a single application in order to see the app messages.
For further testing, you can adjust the cron job's timing to a minute and see it's  operation


## Checklist:
